src/App.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import Navbar from './components/Nav/NavBar';
-import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import { HashRouter, Switch, Route } from 'react-router-dom';
```

Comment:
Can you educate me?  I don't know why BrowseRouter vs. HashRouter.

Reply:
Yup.
We are using different paths in the url to identify the separate pages in the app.  When you are using apache, it is trying to serve up a page called slides.html when you go to slides.  We don't really have a page called that.

In order to avoid that issue until we modify apache to rewrite things to /index.html we are using HashRouter instead of BrowserRouter.

@zwright, we will need to be careful when we write the rewrite rules so that we can still do our rewrite to go to the spring backend with anything going to /api

Reply:
:+1: Thank you, robust answer!